kits/news-digest/prompts/news-digest_llmnode-683_user_1.md

```
@@ -0,0 +1,2 @@
+Topic: {{variablesNode_218.output.topic}}
+Scraped content:{{firecrawlNode_969.output.data}}
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)